modules/virt-verifying-online-snapshot-creation-with-snapshot-indications.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-managing-vm-snapshots.adoc
+
+[id="virt-verifying-online-snapshot-creation-with-snapshot-indications_{context}"]
+= Verifying online snapshot creation with snapshot indications
+
+Snapshot indications are contextual information about online virtual machine (VM) snapshot operations. They are helpful in describing details about the online snapshot creation.
+
+.Prerequisites
+
+* To view indications, you must have attempted to create an online VM snapshot using the CLI or the web console.
+
+.Procedure
+
+. Display the output from the snapshot indications by doing one of the following:
+* For online snapshots created with the CLI, view indicator output in the `VirtualMachineSnapshot` object YAML, in the *status* field.
+* For online snapshots created using the web console, click *VirtualMachineSnapshot > Status* in the *Snapshot details* screen.
```

Comment:
You can remove the word online from the above two entries.  Offline snapshots will currently have no indications listed.

Reply:
Done.